plugins/python/gil.c

```diff
@@ -4,19 +4,19 @@ extern struct uwsgi_server uwsgi;
 extern struct uwsgi_python up;
 
 void gil_real_get() {
-	//uwsgi_log("LOCK %d\n", uwsgi.mywid);
+	uwsgi_log("LOCK %d\n", uwsgi.mywid);
```

Comment:
is this a change that needs to be made?

guessing they want to keep this commented out?

Reply:
Right, it should be commented out but helps in showing when/how workers deadlock.